packages/components/src/InputPhoneNumber/InputPhoneNumber.tsx

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import { FormField, FormFieldProps } from "../FormField";
+
+interface AllowedCountries {
+  NorthAmerica: { countryCode: number; format: string };
+  GreatBritain: { countryCode: number; format: string };
+}
+//TODO decide on this number formatting if it is best for the UK
+const countries: AllowedCountries = {
+  NorthAmerica: { countryCode: 1, format: "(###) ###-####" },
+  GreatBritain: { countryCode: 1, format: "(###) #### ######" },
+};
+
+// should it be form field or common form field?
+interface InputPhoneNumberProps extends FormFieldProps {
+  alwaysShowMask?: boolean;
+  country: keyof AllowedCountries;
+  placeholder?: string;
+  showCountryCode?: boolean;
+}
+
+export function InputPhoneNumber({
+  country,
+  placeholder,
+  alwaysShowMask = true,
+  showCountryCode = false,
+  ...rest
+}: InputPhoneNumberProps) {
+  const [value, setValue] = React.useState("");
```

Comment:
The caller of the InputPhoneNumber should probably be the one who manages the state of `value`, and `onChange`. We don't need to define those in our component.

This input would be consumed like this:

```jsx
function MyEchoComponent() {
  const [value, setValue] = useState("");
  
  return (
    <>
      <InputPhoneNumber onChange={setValue} value={value} />
      <p>Echo: {value}</p>
    </>
  )
}
```

Reply:
Done!